usaspending_api/references/v2/views/cfda.py

```diff
@@ -51,7 +51,7 @@ def get_cfdas(self):
         while not cdfas and remaining_tries:
             remaining_tries -= 1
             response = post(
-                "https://www.grants.gov/grantsws/rest/opportunities/search/cfda/totals",
```

Comment:
Would it be better to make this URL a setting/configuration? Usually data that can change is best stored in settings/configs.

Reply:
Yeah, that's probably a good call. I'm going to close the PR and make the change in the QAT version of this PR